src/shared/models/order-items.model.ts

```diff
@@ -0,0 +1,5 @@
+import { Merch } from "./merch.model";
+export interface orderItems {
+    quantity: number,
+    product: string | Merch
```

Comment:
I think product shape could be product: Merch | Game to order both merch and games but I am not sure if that's possible, something we can discuss maybe

Reply:
I think a little differently than Aman on this one, I say we do not have an order for games and only for merch. So this is fine(string if we are passing the id | merch if populated ), and we purchase each game individually then add it straight onto the user or in a separate collections similar to how we are doing friends

Reply:
Yes exactly but we can talk about it more tomorrow morning